config/env/production.js

```diff
@@ -47,12 +49,20 @@ if (dbURL) {
 }
 
 // If running in Cloud Foundry with an S3 credential service available
-if (AWSCreds) {
-  AWS.config.update({
-    accessKeyId: AWSCreds.access_key,
-    secretAccessKey: AWSCreds.secret_key,
-    region: AWSCreds.region || 'us-east-1'
+if (sqsKey && sqsSecret && AWS_S3_CREDS) {
+  module.exports.SQS = new AWS.SQS({
+    accessKeyId: sqsKey
+    secretAccessKey: sqsSecret,
+    region: 'us-east-1'
```

Comment:
ok, we can env it when it actually makes sense to env it-- no point in changing something when it is unclear when it will be able to have more than one value.